G-licious/Controllers/StationsViewController.m

```diff
@@ -1,405 +0,0 @@
-//
```

Comment:
I figured once we've gotten our data source back, we can roll back master to 62176b4, the commit right before "Remove old G-licious files"